cfpb/cfgov-refresh
# Upgrade cfpb-chart-builder to v5.1.2 to make 5y view the default for Consumer Credit Trends

v5.1.2 release description: https://github.com/cfpb/cfpb-chart-builder/releases/tag/v5.1.2

> - Test IE11 instead of 10 since we don't support IE10 js https://github.com/cfpb/cfpb-chart-builder/pull/166
> - Remove "All" button from bar charts for temporary fix for wrong quarterly data https://github.com/cfpb/cfpb-chart-builder/pull/164


## Additions

-

## Removals

-

## Changes

- Upgrade cfpb-chart-builder to latest version 

## Testing

1. check out branch and run `./frontend.sh`
1. run server `./runserver.sh`
1. go to a CCT page and confirm that bar charts no longer have an "All" button and that all charts have "5y" button selected by default: http://localhost:8000/data-research/consumer-credit-trends/auto-loans/borrower-risk-profiles/

## Screenshots

![screen shot 2019-02-26 at 5 18 52 pm](https://user-images.githubusercontent.com/702526/53450578-9c5d4700-39ea-11e9-9d7d-65d3cf982f82.png)

## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
